test/expect-tests/dune_rpc/dune

```diff
@@ -5,10 +5,12 @@
  (libraries
   ocaml_config
   dune_util
+  dune_console
   dune_rpc_private
   dune_rpc_server
   dune_rpc_impl
   dune_rpc_client
+  dune_engine
```

Comment:
can you separate this into a different test library? Something like `dune_rpc_impl_tests`.

This library is specifically for testing the RPC machinery and not concrete dune rpc things

Reply:
Sure